### DISCUSSION TITLE
Datadog exporter metric doesn't accept IntGuage Type

### DISCUSSION THREAD

**Discussion by parth-purohit** [open-telemetry/opentelemetry-collector-contrib#4669] [Q&A] [answered]
Our receiver is sending IntGuage datatype metrics, Datadog exporter throws error as below:

2021-08-17T07:06:50.659Z	debug	datadogexporter@v0.31.0/metrics_translator.go:271	Unknown or unsupported metric type	{"kind": "exporter", "name": "datadog/api", "metric name": "CountersBitsOut", "data type": "IntGauge"}

Does datadog exporter have plans to support IntGuage, IntSum  in future or is it a bug?

**Comment by mx-psi** [comment, marked as answer]
@parth-purohit thanks for reporting this! This metric type was deprecated and removed (see open-telemetry/opentelemetry-collector#3731), so I am not sure why you are getting this, it is probably a bug on another component in your pipeline. 

Just to confirm, you are using the opentelemetry-collector-contrib distro version 0.31, right? It would also be helpful to know your full pipeline configuration (please remove API keys or other secrets before sharing 😄).

I will have a look but maybe @codeboten can also shed some light into this?
